install/roles/images/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Obtain/Upload guest_images to OpenStack Cloud
```

Comment:
Overall this role feels unrelated to the pbench change, perhaps factor this out into its own PR?

Reply:
This role is needed for uploading the image to glance to spawn a client and server vm for uperf testing